Config/Global.xcconfig

```diff
@@ -17,6 +17,7 @@ PRODUCT_NAME = $(PROJECT_NAME)
 PUBLIC_HEADERS_FOLDER_PATH = include/$(PRODUCT_NAME)
 
 // :Mark: Language configurations
+
```

Comment:
Is it normal not to have the `CLANG_MODULES_AUTOLINK = NO` here?

Reply:
We don't need that because we stay to build the SDK on Xcode 8.

Reply:
@TroubleMakerBen Probably you can remove the empty line. 😄

Reply:
OK so we Bitrise is back to XCode 8 I guess.